src/number.js

```diff
@@ -26,6 +26,33 @@ define([
 	numberFormatterFn, numberFormatProperties, numberNumberingSystem, numberParserFn,
 	numberParseProperties, numberPattern, numberSymbol, stringPad ) {
 
+function validateDigits( properties ) {
+	var minimumIntegerDigits = properties[ 2 ];
+	var minimumFractionDigits = properties[ 3 ];
+	var maximumFractionDigits = properties[ 4 ];
+
+	var minimumSignificantDigits = properties[ 5 ];
+	var maximumSignificantDigits = properties[ 6 ];
```

Comment:
For example:

https://github.com/jquery/jquery-ui/commit/803eaf29f732a68793b806356096fe849b9f470b#diff-29e4c930be69ff962e7aee3f71878229R144
https://github.com/jquery/qunit/commit/eaae69b822cb1410bd4c61eedd1b4a120757a5fe#diff-586f09c9d3a303fcf4f478bd0b1d5ca4R1